src/lazyload-image.directive.ts

```diff
@@ -24,7 +24,7 @@ interface LazyLoadImageDirectiveProps {
     lazyImage: string;
     defaultImage: string;
     errorImage: string;
-    scrollTarget: HTMLElement;
+    scrollTarget: any;
```

Comment:
This type could have been unchanged but I guess it could be good to be consistent.

Reply:
Hmm, you're right. But I agree that consistency should be maintained, at least within the same file.